### PR TITLE
chore: add new video on Copilot and other ways to deploy to AWS

### PR DESCRIPTION
**Pull request by Fodoj**
I've made a video about some of the existing tools to deploy to AWS, with a conclusion that Copilot is the best among them right now - and a demo of Copilot's feature, applied to a real application I've recently developed and released :-)

### CODE REVIEW

**Review by iamhopaul123 — APPROVED** [review]
Woohoo this is fantastic 🔥 Thank you so much for making this video!

**Comment by huanjani** [comment]
Wow! This is so great! Thanks for adding the link to our docs. 
Just one tiny nit.

Also, to pass our picky checks, would you mind amending your commit msg so it follows conventional commits? Thank you!

File: site/content/community/guides.en.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Share your applications, videos, and blog posts with fellow Copilots!
 | Title      | Description                          |
 | ----------- | ------------------------------------ |
 | [**Containers from the Couch series**](https://www.youtube.com/c/ContainersfromtheCouch/search?query=copilot) by <a href="https://twitter.com/realadamjkeller">@realadamjkeller</a>, <a href="https://twitter.com/brentContained">@brentContained</a>, and guests | Join Adam and Brent to learn about all the existing features of AWS Copilot with fun demos. From setting up a three-tier application with autoscaling to creating a continuous delivery pipeline with integration tests. |
+| [**AWS Copilot and another 6 ways to easily deploy apps to AWS**](https://www.youtube.com/watch?v=0lxcfJ20TwA) by <a href="https://twitter.com/Fodoj">Kirill Shirinkin</a> | Kirill Shirinkin explores the ways to easily deploy new product on AWS - including a new favourite one, AWS Copilot. |
```
Comment (huanjani) [inline site/content/community/guides.en.md]: Nit: I think we should probably spell "favorite" the American way for consistency with the rest of the docs. 
```suggestion
| [**AWS Copilot and another 6 ways to easily deploy apps to AWS**](https://www.youtube.com/watch?v=0lxcfJ20TwA) by <a href="https://twitter.com/Fodoj">Kirill Shirinkin</a> | Kirill Shirinkin explores ways to easily deploy a new product on AWS - including a new favorite one, AWS Copilot. |
```

**Comment by Fodoj** [comment]
@huanjani thanks! I've rebased and applied your fix :)

**Review by huanjani — APPROVED** [review]
Awesome! Thank you!

**Comment by efekarakus** [comment]
Awesome, the changes look great I think for Semantic Pull Request to pass you might need to rename the commit title to `docs: add new video on Copilot`. These are the allowed types (before the `:`) for conventional commits: https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum

**Pull request merged**